buildout.cfg

```diff
@@ -92,8 +92,10 @@ cmds =
     >>>                     print line
     >>> fileinput.close()
 
-[liblas]
+[laspy]
 recipe = collective.recipe.cmd
 on_install = true
 on_update = true
-cmds = ${buildout:directory}\buildout\bin\pip.exe install ${buildout:directory}\wheels\${vars:liblas}
+cmds = ${buildout:directory}\buildout\bin\pip.exe install https://github.com/sitn/laspy/archive/e2a1a3307d4c252f80855c3e80ec373fdfc7265b.zip
```

Comment:
I really do not get why during the installation it tries to install/reach something in your Python root folder... We are using a virtual environment, so it should touch these folders....